bin/postgres_common/postgres/modules/pgmonitor.sh

```diff
@@ -19,39 +19,39 @@ then
     then
```

Comment:
pgmonitor.sh exists in both postgres and postgres-ha variants - need to figure out if I can combine the two

Reply:
I believe the `postgres-ha` variant was set up for the newer monitoring work. I believe we'd want to use that and apply it back to whatever is in `postgres`. Basically, take the most up-to-date one and combine it.